aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. 
+
+* How to get started.
+* The benefits and potential drawbacks.
+* The limitations.
```

Comment:
I wonder if we want to introduce what native AOT is and what benefits it has before "How to get started.". In my mind the flow is:

1. introduce the concept
2. describe the pros and cons
3. describe how to use it

Reply:
In the opening paragraph we have a link to the native AOT deployment page which I think gives  pretty good coverage. I was trying to avoid duplicating that content.

In terms of sequencing of prons/cons vs. a basic how to, I generally favor putting tyre kicking content up front. It gives folks a food idea of the basic mechanics (e.g. AOT happens during publish). I initially had it around the way you suggested but then I think @Rick-Anderson made an edit which made it more logical for it to go first (which I agreed with).